Application/Contracts/MovieFilters.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Application.Contracts;
+
+public class MovieFilters
```

Comment:
I would name this as MovieFilter.
![image](https://github.com/andjela11/movie/assets/129784988/1c10c658-8822-4884-a31b-6404464a5f08)